src/components/App.js

```diff
@@ -0,0 +1,74 @@
+import { request } from "../api/api.js";
+import DocumentPage from "./DocumentPage.js";
+import DocumentEditPage from "./DocumentEditPage.js";
+import { initRouter } from "../routes/router.js";
+
+export default function App({ $target }) {
+  this.state = {
+    docId: null,
+    docs: [],
+  };
+
+  this.setState = (nextState) => {
+    this.state = nextState;
+    documentPage.setState(nextState);
```

Comment:
```suggestion
    documentPage.setState(this.state);
```
`this.state`가 `nextState` 값이기 때문에 `setState(this.state)`가 적절할 것 같습니다!! 😁